src/librustdoc/clean/types.rs

```diff
@@ -1405,6 +1405,11 @@ impl Path {
     pub fn last_name(&self) -> &str {
         self.segments.last().expect("segments were empty").name.as_str()
     }
+
+    pub fn whole_name(&self) -> String {
```

Comment:
Is this contextually dependent? What's the difference towards a fully qualified path?

Reply:
Shouldn't be contextually dependent, it _is_ just the fully qualified path.